modules/app_lb/main.tf

```
@@ -3,6 +3,36 @@ locals {
   https_port = 443
 }
 
+resource "aws_security_group" "inbound-private" {
+  count       = var.enable_private_only_traffic ? 1 : 0
+  name        = "${var.namespace}-nlb-inbound-private-endpoint"
```

```suggestion
  name        = "${var.namespace}-nlb-inbound-private-endpoint"
```

whats the name length limit? This might cause issues for our dedicated envs

---

up to 255 characters